src/gui_common/tooltip/ToolTipManager.cs

```diff
@@ -0,0 +1,262 @@
+using System.Collections.Generic;
+using Godot;
+
+/// <summary>
+///   Holds and handles a collection of custom tooltip Controls
+/// </summary>
+public class ToolTipManager : CanvasLayer
```

Comment:
I think the layer is unset for this? I think the tooltip layer should be set to 3